docs/contributing/overview.md

```diff
@@ -1 +1,53 @@
-This section goes deep explaning Tracee's internals, and how to build Tracee on different platforms for the users interested in contributing.
+## Contributing
+
+Thank you for taking interest in contributing to Tracee! This document covers our working practices and conventions.
+
+## Issues and Discussions
+
+We encourage open discussion and collaboration using both GitHub Issues and Discussions.  
+
+- [Discussions](https://github.com/aquasecurity/tracee/discussions) are free-style conversational tool, we use them for conversations.
+- [Issues](https://github.com/aquasecurity/tracee/issues) are project management tool, we use them to keep track on who's working on what and plan ahead.
+
+If you have a suggestion, question, or a general comment - please use Discussions. If there's a clear work item (including bugs) - you can open an Issue.
+
+### Discussions:
+
+1. We have the following discussion topics: 
+  1. [Announcements](https://github.com/aquasecurity/tracee/discussions/categories/announcements): One way communication from the team to the community. Consider this like our mini blog
+  2. [Questions and Help](https://github.com/aquasecurity/tracee/discussions/categories/questions-and-help): For help and support. Consider this similar to StackOverflow.
+  3. [Development](https://github.com/aquasecurity/tracee/discussions/categories/development): For discussing potential features, and collaborating on their design.
+
+### Issues:
+
+1. Every issue needs to be actionable and assignable. Consider the scope of the issue if assigned to one person, and break down if necessary.
```

Comment:
the points here don't seem to be working properly? i.e. 
1. ....
1. ....
     1.
     2.

Reply:
I haven't checked but I would like to raise awareness for: bullets/numbers for markdown are more complete than from mkdocs. I had to make sure that both worked when I did the initial documentation (and the end of the lines, for mkdocs, had to have 2 spaces "  " for line breakage). So there are some caveats and we have to make sure both are good: markdown and mkdocs. My 5 cents only.

Reply:
Good call out. I fixed the issue here (and elsewhere in the file) and verified that it rendered well on both github and in mkdocs. Interestingly, the sublists render as numbers on github and letters in mkdocs. Either way looks good.